src/SwiftTaskProvider.ts

```diff
@@ -39,10 +39,15 @@ interface TaskConfig {
 
 /**
  * Creates a {@link vscode.Task Task} to build all targets in this package.
- * This excludes test targets.
  */
 function createBuildAllTask(): vscode.Task {
-    const additionalArgs = (process.platform !== 'darwin') ? ['--enable-test-discovery'] : [];
+    const additionalArgs: string[] = [];
+    if (process.platform !== 'darwin') {
+        additionalArgs.push('--enable-test-discovery');
+    }
+    if (process.platform === 'win32') {
+        additionalArgs.push('-Xlinker', '-debug:dwarf');
+    }
```

Comment:
Please expose this as an option.  This guarantees a ~90% chance of completely disabling debugging abilities (if you have a mixed codebase).  Codeview doesn't let you always debug Swift code, but for mixed codebases, it is preferred, and when it works, is more stable than DWARF.  It does however preclude the use of lldb.